front/app/modules/commercial/insights/admin/containers/Insights/Details.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import TopBar from '../../components/TopBar';
+import { withRouter } from 'react-router';
+
+const DetailsInsightsView = () => {
+  return (
+    <>
+      <TopBar />
+    </>
```

Comment:
Do we need the fragment tags here? (Doesn't really matter, just curious why if we do)

Reply:
Not right now, but we will as the page start to fill out. Will remove it for now :)